website/docs/FAQ.md

```diff
@@ -127,3 +127,34 @@ Workaround:
 2. `mkdir /home/vscode/.local/lib/python3.10/site-packages/google/colab`
 
 Explanation: Per [this gist](https://gist.github.com/defulmere/8b9695e415a44271061cc8e272f3c300?permalink_comment_id=4711478#gistcomment-4711478), linked from the official [chromadb docs](https://docs.trychroma.com/troubleshooting#sqlite), adding this folder triggers chromadb to use pysqlite3 instead of the default.
+
+## How to register a reply function
+
+(from [issue #478](https://github.com/microsoft/autogen/issues/478))
+
+You can hook in a function that is called whenever an agent receives a message
```

Comment:
The registered reply function is only called when `generate_reply` is called. If an agent receive a message but a reply is not requested, it won't be called.